manim/default.cfg

```diff
@@ -88,6 +88,10 @@ frame_rate = 60
 pixel_height = 1440
 pixel_width = 2560
 
+max_files_cached = 7
```

Comment:
Suggestion : Can we have the default number of `max_files_cached` be infinity? I personally think it would be much more convenient that way.  We could set it to `-1` here, like we do for `upto_animation_number`.

Reply:
I partially disagree. It should not be infinity by default as it would very quickly become a huge mess. Maybe should we raise the default value to 15, I don't know (which is largely enough, I think). 
But I will as the possibility to set an infinity value. Thank you for the suggestion !

Reply:
No, even 15 is not enough. If it is used for making video, it is impossible to have only 15 animations in each `Scene` including `wait`. If it is set to such a small value, it will not have any cache effect. I think it is most appropriate to set it to 70~100, which can have a cache effect and the memory is not too much (not even more than 30MB). Moreover, in some cases, a `Scene` may contain thousands of animations.

Reply:
This is a normal `Scene` of a normal video:
![example1](https://user-images.githubusercontent.com/44120331/89028666-6a7aad80-d35f-11ea-808c-8d711b39e3bb.png)
This is a complex Scene made by for, and the video is only 3 minutes and 10MB:
![example2](https://user-images.githubusercontent.com/44120331/89028804-a0b82d00-d35f-11ea-8085-05be5e385ce6.png)

Reply:
@Tony031218 I hear that, but I don't think you represent the average user of manim. 70-100 is for me way way way too much, but I don't represent as well either the average user of manim as I don't have much experience with doing scenes.

I think I will conduct a survey on the discord server, and/or open an issue about that :D

Reply:
This is not the case. The 70~100 I mentioned is relatively small, and this is not an exaggeration. If you use manim to make a complete video, a Scene requires a lot of animations, and even if you keep 100 cached videos, it takes up space It will not exceed 5MB.